examples/llama/README.md

```diff
@@ -1,9 +1,41 @@
 # Llama
 
-## Get model
+## Clone this repo
```

Comment:
We can give user another option to pip install from pypi once we have python package

Reply:
Oh yeah for sure. I wanted to make sure it is correct for what we have currently.